examples/git_tester.py

```diff
@@ -1,34 +1,44 @@
+import os
+
+from sys import platform
 from pydriller import Repository
-import os 
-import datetime
-os.system("cp examples/speedtest.py examples/speedtest2.py") # the file has to be outside of git
-for idx, commit in enumerate(Repository('.', from_tag="v0.6.0").traverse_commits()):    
-    name=commit.msg.replace('\n', ' ').replace('\r', ' ')
-    print(idx, commit.hash, name)
 
+if platform == "win32":
+    copy_cmd = "copy"
+    rm_dir_cmd = "rmdir /s /q"
+else:
+    copy_cmd = "cp"
+    rm_dir_cmd = "rm -rf"
 
+speedtest_src_path = os.path.join("examples", "speedtest.py")
+speedtest_copy_path = os.path.join("examples", "speedtest2.py")
+os.system(f"{copy_cmd} {speedtest_src_path} {speedtest_copy_path}")  # the file has to be outside of git
```

Comment:
Maybe use `shutil.copyfile`and `shutil.rmtree` instead of checks?
I also think "/" works in windows as well.

Reply:
Added shutil.
os.path.join is a standard way of combining paths. I'd like to keep it.